tests/resources/crds/test.run_clustertestobjs.yaml

```diff
@@ -0,0 +1,145 @@
+# Copyright 2021 VMware
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: clustertestobjs.test.run
+spec:
+  group: test.run
+  names:
+    kind: ClusterTestObj
+    listKind: ClusterTestObjList
+    plural: clustertestobjs
+    singular: clustertestobj
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
not really very related to the PR, but, I think we could generate these without description being set - passing ` crd:maxDescLen=0` to `controller-gen` would do it

Reply:
(I think we could go with no description for the `test` ones, but then keep description for the non-test ones)